backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryParser.cs

```diff
@@ -88,6 +88,11 @@ public virtual ValueTask<Q> ParseAsync(Context context, Q q, ISchemaEntity? sche
                 {
                     query.Take = options.DefaultPageSize;
                 }
+
+                if (q.CreatedBy != null)
```

Comment:
I think this is not needed, looks wrong in my opinion.

Reply:
If I delete this, user gets all contents.  But he must get only own contents.